sys/fuchsia/fidl.txt

```diff
@@ -23,9 +23,8 @@ type fidl_union_member[TAG, TYPE] {
 
 type fidl_message_header[METHOD_ORDINAL] {
 	txid		const[0, int32]
```

Comment:
```
A value of 0 for txid is reserved for messages which do not require a response from the other side. Note: For more details on txid allocation, see zx_channel_call().
```

Does it affect us in any way? Or nobody really checks/enforces this convention?

Reply:
I'm not sure. It didn't caused a problem for our minimal test case at least. I'll keep that in mind going forward though.